Bio/SearchIO/BlatIO.py

```diff
@@ -653,7 +656,8 @@ def _build_row(self, qresult):
                 line.append(hsp.hit_gap_num)
 
                 # check spans
-                assert hsp.query_span_all == hsp.hit_span_all
+                eff_query_spans = [blocksize_multiplier * s for s in hsp.query_span_all]
+                assert hsp.hit_span_all == eff_query_spans
```

Comment:
There are probably lots of asserts in Biopython which might be better done as explicitly exceptions, don't let this delay merging this pull request.